app/utils/images/index.ts

```diff
@@ -2,6 +2,7 @@
 // See LICENSE.txt for license information.
 
 import {Dimensions} from 'react-native';
+import 'react-native-reanimated';
```

Comment:
Just to understand, this is needed so the worklet defiend later works correctly, right?